client/src/components/AttendeeInput.tsx

```diff
@@ -33,44 +32,24 @@ export default function AttendeeInput({ id, onChange, value, type }: AttendeeInp
 
   const handleSelectionChange = (_: React.SyntheticEvent, newValue: Array<string | IPeopleInformation>) => {
     const emails = newValue.map((option) => (typeof option === 'object' && option.email ? option.email : (option as string)));
-    const filteredEmails = emails.filter((email) => emails.indexOf(email) === emails.lastIndexOf(email));
-    if (filteredEmails.length > 0) {
-      const lastValue = filteredEmails[filteredEmails.length - 1].trim();
-      if (isEmailValid(lastValue)) {
-        onChange(id, filteredEmails);
-        setTextInput('');
-      } else {
-        toast.error('Invalid email entered');
-      }
-    } else {
-      onChange(id, filteredEmails);
-      setTextInput('');
+    const filteredEmails = emails
+      .join(' ')
+      .split(/\s+/)
+      .map((email) => email.trim())
+      .filter((email) => email !== '');
+    const uniqueEmails = [...new Set(filteredEmails)];
+    const validEmails: string[] = [],
+      invalidEmails: string[] = [];
```

Comment:
assignment of variables with comma separation makes it hard to debug code. It's a general practice to do 1 declaration/assignment per line for readability.